src/SearchBox.jsx

```diff
@@ -30,8 +30,21 @@ function SearchBox({ setData, setError }) {
 
     if (selectedOption === "titolo" && titolo) queryString += "&ti=" + titolo;
     if (testo) queryString += "&txt=" + testo;
-    if (selectedOption === "area-tecnica" && areaTecnica)
-      queryString += "&tecarea=" + areaTecnica;
+    if (selectedOption === "area-tecnica" && areaTecnica) {
+      switch (areaTecnica) {
+        case "freno":
```

Comment:
:) this solution will work for now. So let's keep it as a "quick and dirty" one :)
But in the future (hopefully) the values of "area-tecnica" and even of "titolo" will be "customer-specific" and you cannot hard-code them like this.